.github/workflows/prod-sync-artifacts.yml

```diff
@@ -24,6 +26,7 @@ jobs:
         run: ./release-tools/scripts/required_packages.sh
       - name: Install python packages
         run: |
+          sudo apt-get install python-setuptools
```

Comment:
This should be included in the required_packages.sh since it is ubuntu specific.

Reply:
Added in the required packages.